### PR TITLE
[WONTFIX] Add support for JUnitXML Test logger in Sphinx-Gallery & CircleCI

### PR DESCRIPTION
**Pull request by kchawla-pi**
Log info about build times

### CODE REVIEW

**Comment by kchawla-pi** [comment]
@GaelVaroquaux Review this please if you get a chance. I don't see any test-results unloaded as artifacts.
Does that happen only when running a full build?
If so, review this and I will merge it , then we will see.

**Comment by bthirion** [comment]
LGTM.

**Comment by kchawla-pi** [comment]
Yeah except I don't see any report after circle CI has run.
Dunno yet how to fix it.

File: .circleci/config.yml [hunk 1]
```diff
@@ -95,7 +95,11 @@ jobs:
             source activate testenv
             pip install -e .
             set -o pipefail && cd doc && make html-strict 2>&1 | tee log.txt
+            mkdir -p _build/test-results
           no_output_timeout: 5h
+      - store_test_results:
+          path: doc/_build/test-results
```
Comment (effigies) [inline .circleci/config.yml]: Possibly this needs to be an absolute path?

**Comment by kchawla-pi** [comment]
This is not a priority anymore. The nistats project will soon be merged into Nilearn, which already has this functionality.

**Pull request closed without merging**